server/src/main/java/com/jungleapp/cs414/server/JungleBoard.java

```diff
@@ -110,6 +110,21 @@ boolean makeMove(int row, int column, int toRow, int toColumn) {
         return false;
     }
 
+    boolean zeroPieces(String color) {
+        int numPieces = 0;
+        for(int i = 0; i < board.length; i++) {
+            for(int j = 0; j < board[i].length; j++) {
+                if(board[i][j] != null) {
+                    if(board[i][j].getColor().equals(color)) {
```

Comment:
Would it be better to just return False inside of this if statement, instead of counting all of the pieces, since we only need to find one piece of a certain color. Return True if it never goes in to the if statement. This way we could avoid looking through the whole board every time.

Reply:
That's a good, call. I'll update that.